FieldOpt/Reservoir/grid/cell.cpp

```diff
@@ -22,20 +22,25 @@
 namespace Reservoir {
     namespace Grid {
 
-
-        Cell::Cell(int global_index, IJKCoordinate ijk_index,
-                   double volume, double poro, double permx, double permy, double permz,
-                   Eigen::Vector3d center, std::vector<Eigen::Vector3d> corners)
+        Cell::Cell(int global_index,
+                   IJKCoordinate ijk_index,
+                   double volume,
+                   double poro,
+                   double permx,
+                   double permy,
+                   double permz,
+                   Eigen::Vector3d center,
+                   std::vector<Eigen::Vector3d> corners)
         {
             global_index_ = global_index;
-            ijk_index_ = ijk_index;
-            volume_ = volume;
-            porosity_ = poro;
-            permx_ = permx;
-            permy_ = permy;
-            permz_ = permz;
-            center_ = center;
-            corners_ = corners;
+               ijk_index_ = ijk_index;
+                  volume_ = volume;
+                porosity_ = poro;
+                   permx_ = permx;
+                   permy_ = permy;
+                   permz_ = permz;
+                  center_ = center;
+                 corners_ = corners;
```

Comment:
I don't like this indentation. In general, if the auto indentation in the IDE (<kbd>ctrl</kbd>+<kbd>alt</kbd>+<kbd>i</kbd>) changes the indentation, it's not the way it should be (with the exception of some comments).